app/assets/javascripts/actions/story.js

```diff
@@ -95,6 +96,20 @@ export const setLoadingStory = (id, from) => ({
   from
 });
 
+export const closeEpic = () =>
```

Comment:
I think it doesn't need to be a thunk, right? We can do like this:

```js
export const closeEpic = () => receiveStories([], storyScopes.EPIC)
```

Actually, it would be even better to create an actual action `CLOSE_EPIC` and handle it in stories reducer directly instead of reusing this action like this.

Reply:
Sure!